tools/ci_build/github/android/default_qnn_aar_build_settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "build_abis": [
+        "arm64-v8a"
+    ],
+    "android_min_sdk_version": 21,
+    "android_target_sdk_version": 24,
+    "build_params": [
+        "--enable_lto",
+        "--android",
+        "--parallel",
+        "--cmake_generator=Ninja",
+        "--build_java",
+        "--build_shared_lib",
+        "--use_qnn",
+        "--skip_tests"
```

Comment:
If we're skipping the tests we could add this to not build them at all (speeds up builds a lot)

`"--cmake_extra_defines=onnxruntime_BUILD_UNIT_TESTS=OFF"`